src/styles/style.css

```diff
@@ -0,0 +1,122 @@
+@import "./star.css";
+
+:root {
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+
+  /* button bg and :hover border+text color */
+  --button-bg-color: #00acdc;
+
+  --button-text-color: #fff;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
i'd rather put these styles in another file like normalize.css. this is why my own task was rejected

Reply:
Strongly agree, will do that!
Thanks!